nipype/tests/test_utils.py

```diff
@@ -0,0 +1,12 @@
+from nipype import utils
+import pytest
+
+
+@pytest.mark.parametrize("load_versioning", [True, False])
+@pytest.mark.parametrize("save_versioning", [True, False])
+def test_pickle(tmp_path, save_versioning, load_versioning):
```

Comment:
Oh, this test should go in `nipype/utils/tests/test_filemanip.py`. Sorry, I just caught this.

Reply:
Oops I didn't see that file.